docs/Admin.md

```diff
@@ -595,6 +598,44 @@ const App = () => (
 );
 ```
 
+### Unplug the <Admin> using the `<AdminContext>` and the `<AdminUI>`
+
+Setting Resources dynamically is still very cumbersome: even if `<Admin>` accepts [a function as child](#declaring-resources-at-runtime), this function can't execute hooks.
```

Comment:
```suggestion
Setting Resources dynamically using the children-as-function syntax may not be enough in all cases, because this function can't execute hooks.
```